tests/test_routes.py

```diff
@@ -65,6 +64,38 @@ def tearDown(self):
     #  C U S T O M E R S   A P I   T E S T   C A S E S
     ######################################################################
 
+    ######################################################################
+    #  A C T I V A T E / D E A C T I V A T E   C A S E S
+    ######################################################################
+
+    def test_activate_customer(self):
+        """ It should activate customer """
+        # Create dummy deactivated customers
+        customers = CustomerFactory.create_batch(3)
+        
+        for customer in customers:
+            customer.active=False
+            customer.create()
+            
+        # choose a customer id randomly to delete
+        all_customers = Customer.all()
+        self.assertEqual(len(all_customers), 3)
+
+        # pick random customer
+        rand_customer = random.choice(all_customers)
+        self.assertFalse(rand_customer.active)
+        rand_customer_id = rand_customer.id
+
+        # api call to activate the chosen customer
+        resp = self.client.put(f"{BASE_URL}/{rand_customer_id}/activate")
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
```

Comment:
We should add the assertion for active field here after calling activate.
```
        self.assertEqual(rand_customer_id, activated_customer["id"])
        self.assertTrue(activated_customer["active"])
```

Reply:
Done, please check.